src/ManagerStandalone.sol

```diff
@@ -29,6 +29,13 @@ contract ManagerStandalone is IManagerStandalone, Manager, Implementation {
         _checkEndpointsInvariants();
     }
 
+    /// @dev Override the [`renounceOwnership`] function to ensure
+    /// the manager ownership is not renounced.
+    function renounceOwnership() public override onlyOwner {
+        // do nothing
+        // this method body is empty
+    }
```

Comment:
should we revert here instead of doing nothing?

Reply:
Agree re reverting on all of these overrides